save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsOrganizationsMenuView.kt

```diff
@@ -26,6 +27,16 @@ import react.dom.html.ReactHTML.ul
 @Suppress("MISSING_KDOC_TOP_LEVEL", "TOO_LONG_FUNCTION", "LongMethod")
 class UserSettingsOrganizationsMenuView : UserSettingsView() {
     private val organizationListCard = cardComponent(isBordered = false, hasBg = true)
+    private val comparator: Comparator<OrganizationWithUsers> =
+            compareBy<OrganizationWithUsers> { orderedOrganizationStatus[it.organization.status] }
+                .thenBy { it.organization.name }
+    private fun localButtonAction(organizationWithUsers: OrganizationWithUsers, newStatus: OrganizationStatus) {
+        setState {
+            selfOrganizationWithUserList = selfOrganizationWithUserList.minusElement(organizationWithUsers)
```

Comment:
you can use `TreeSet` with custom `comparator` instead of List and sorting it

Reply:
```unresolved references``` in ```frontend```